src/component/LazyPage2.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { lazy, Suspense } from 'react';
 
-const Page2 = lazy(() => import('./Page2'));
+const Page2 = lazy(() => import('./SignIn'));
```

Comment:
File name is `src/component/LazyPage2.tsx`
Import is `./SignIn`
Variable name is `Page`

Very nice.